infra/terraform/terraform.go

```diff
@@ -314,7 +323,7 @@ func (r *terraform) command(ctx context.Context, args []string, opts ...system.C
 		))
 	err := system.ExecL(cmd, &out, r.Entry, opts...)
 	if err != nil {
-		return out.Bytes(), trace.Wrap(err, "command %q failed (args %q, wd %q)", cmd.Path, cmd.Args, cmd.Dir)
+		return out.Bytes(), trace.Wrap(err, "command %q failed (args %q, wd %q), stdin/stdout: %v", cmd.Path, cmd.Args, cmd.Dir, out.String())
```

Comment:
I think this should be `stdout/stderr` but even this is superfluous. Why not simply output it after a semicolon?